inst/stan/stan_docs/mainpage.md

```diff
@@ -0,0 +1,55 @@
+# Primary Censored Stan Functions
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+The primarycensored R package aims to address common challenges in estimating delay distributions. It does this by providing tools to manipulate primary censored delay distributions in Stan and R.
+
+This documentation covers the Stan functions. All the Stan functions have a corresponding R function, which can be found in the [primarycensored R Reference](https://primarycensored.epinowcast.org/dev/reference/index.html).
+
+## Documentation Structure
+
+Each function page includes:
+- A brief description of each function's purpose
+- Parameter descriptions
+- Return values
+- Grouping structure
+- Example usage where applicable
+- Call and caller graphs for learning the relationships between functions
+
+## License
```

Comment:
You have the license above in a badge as well so likely should pick just one. I think I would take the approach that doesn't requiring copying in

Reply:
Agreed. I've removed the section in 7ede7ff2750fe3563e2b04f926ef565e7edee3ce.